pkg/scm/scm_test.go

```diff
@@ -14,7 +14,12 @@ func createLocalGitDirectory(t *testing.T) string {
 	if err != nil {
 		t.Error(err)
 	}
-	os.Mkdir(filepath.Join(dir, ".git"), 0600)
+	os.MkdirAll(filepath.Join(dir, ".git/refs/heads"), 0777)
+	os.MkdirAll(filepath.Join(dir, ".git/refs/remotes"), 0777)
+	os.MkdirAll(filepath.Join(dir, ".git/branches"), 0777)
+	os.MkdirAll(filepath.Join(dir, ".git/objects/foo"), 0777)
+	os.Create(filepath.Join(dir, ".git/objects/foo") + "bar")
+	//os.Create(filepath.Join(dir, ".git/refs/heads") + "master")
```

Comment:
Thanks for the feedback!  I forgot to take this out.  Since we already create a `bar` object, it is a "valid" git repository, because that would represent a commit.  I'll add a comment explaining that